fast-pysf/pysocialforce/ped_population.py

```
@@ -0,0 +1,295 @@
+        return (v[0] * f, v[1] * f)
+
+    def clip_spread(v):
+        return np.clip(v, -sidewalk_width / 2, sidewalk_width / 2)
+
+    center = add_vecs(start, scale_vec(sub_vecs(end, start), sec_offset / sec_len))
+    std_dev = sidewalk_width / 4
+
+    x_offsets = clip_spread(np.random.normal(center[0], std_dev, (num_samples, 1)))
+    y_offsets = clip_spread(np.random.normal(center[1], std_dev, (num_samples, 1)))
+    points = np.concatenate((x_offsets, y_offsets), axis=1) + center
+    return [(x, y) for x, y in points], sec_id
```

Pylint (4.0.2)

[refactor] Unnecessary use of a comprehension, use list(points) instead.

(R1721)